e2e/features/case-progression.feature

```diff
@@ -159,7 +159,7 @@ Feature: Case progression
     And within the `Upload Home Office bundle` collection's first item, I should see `This is the respondent evidence` in the `Describe the document` field
 
     When I click the `Upload` button
-    Then I should see the text `You've uploaded the Home Office bundle`
```

Comment:
This is the right value if I am remember correctly. The below was an old value. Could you please check?